package/cluster/network/definition.yaml

```diff
@@ -21,6 +21,47 @@ spec:
                 id:
                   type: string
                   description: ID of this Network that other objects will use to refer to it.
+                parameters:
+                  type: object
+                  properties:
+                    deletionPolicy:
+                      type: string
+                      default: Delete
+                      description: Delete or Orphan
+                      enum:
+                        - Delete
+                        - Orphan
+                  region:
```

Comment:
`region` should be at the same indentation level as `deletionPolicy` this prevents the XNetwork from rendering.

Reply:
adjusted.